src/tests/main.spec.js

```diff
@@ -0,0 +1,127 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import faker from 'faker';
+import { days, defaultProduct, money, users } from './constants';
+import {showMessage, getDay, getAdultUsers, getRandomUsers, Product} from './main';
+
+const {assert, expect} = chai;
+chai.should();
+
+describe('showMessage()', () => {
+  const fakeText = 'faker.address.city';
+
+  afterEach(() => {
+    window.alert.restore();
+  });
+
+  it('alert should be invoked', () => {
+    const spy = sinon.spy(window, 'alert');
+    showMessage()
+
+    expect(spy.called).true;
+  });
+
+  it('alert should be invoked with argument', () => {
+    const spy = sinon.spy(window, 'alert');
```

Comment:
the same is here - http://i.imgur.com/n7TFumb.png
spy doesn't prevent an execution of real method